### DISCUSSION TITLE
SABnzbd not merging Segments

### DISCUSSION THREAD

**Discussion by Endthal** [anasty17/mirror-leech-telegram-bot#1720] [Q&A] [answered]
When I download a file (video for example) with SABnzbd, it downloads the RAR segments (like part1.rar, part2.rar, etc.) but doesn't merge them into an MP4 file and uploads all these files directly instead. This happens when uploading to the cloud and to Telegram. However, downloading the same NZB file with SABnzbd directly on my system results in a properly merged mp4 file.

**Comment by anasty17** [comment, marked as answer]
Which arg for extracting ?

**Reply by Endthal** [reply]
ohh now its working. i thought i dont need an arg, but also tried it with -j. didn't think of -e. thanks!
